cmake/modules/Finduring.cmake

```diff
@@ -0,0 +1,21 @@
+# - Find
```

Comment:
might want to drop the comment or to complete it.

Reply:
I'lll complete the comment